server/scripts/download_batch_output.py

```diff
@@ -14,7 +14,7 @@
 downloaded=[]
 
 # Enter the correct timestamp file of the last run batch jobs
-time_stamp="2024-11-04_20:53:24.txt"
+time_stamp="2024-11-25_12:00:55.txt"
```

Comment:
Why is this required?

Reply:
The scripts as a downloader of the correct batch files. I use the timestamp in a previous script to save the batch ids and their dbs. I use this timestamp in the downloader as an indentifier to download the correct batches.

Reply:
Why timestamps? Cant we just generate a UUID as a tracker

Reply:
We could use a UUID but this way it's a little easier to see which batch was run when.
Also we used this convention in all bird experiments and the batch_metadata is stored against this convention for all experiments so I would not recommend changing the convention now.